easybuild/easyconfigs/p/PyBERT/PyBERT-1.7.9_2to3.patch

```diff
@@ -0,0 +1,415 @@
+# Python 2to3
```

Comment:
@boegel: I mean the GUI starts, but I do not know anything about its functionality.